src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -87,6 +112,19 @@ def __call__(self, organization: Organization, event: Mapping[str, Any]):
         except Repository.DoesNotExist:
             raise Http404()
 
+        if "webhook_secret" in repo.config:
```

Comment:
why are we storing the webhook secret in each repository? should the webhook secret be for each integration instance, like we have for gitlab? https://github.com/getsentry/sentry/blob/50658afd7bf8df5f3b25ffe95c4860b81e0d4968/src/sentry/integrations/gitlab/webhooks.py#L317-L322